content/@i18n/ja/concepts/tokens/decentralized-exchange.md

```diff
@@ -1,5 +1,5 @@
 ---
-html: decentralized-exchange.html
+html: de-index.ja.html
```

Comment:
This line should not be changed, and html filenames should _never_ have `.ja.html` in them. Also, all the other pages that linked to this page should also not have their `parent:` attributes changed.

Reply:
Reverted (local builds won't work).